gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -458,6 +462,26 @@ def main(args):
                 cochran_mantel_haenszel_test=res.cmh_ht.ht()[
                     ht.key
                 ].cochran_mantel_haenszel_test,
+                join_metric_data_type=hl.case()
+                .when(
+                    (hl.is_defined(ht.genomes_grpmax.AC))
+                    & hl.is_defined(ht.exomes_grpmax.gnomad.AC),
+                    "both",
+                )
+                .when(hl.is_defined(ht.genomes_grpmax.AC), "genomes")
+                .when(hl.is_defined(ht.exomes_grpmax.gnomad.AC), "exomes")
+                .default(hl.missing(hl.tstr)),
+                joint_fafmax=ht.joint_fafmax.annotate(
+                    join_metric_data_type=hl.case()
+                    .when(
+                        (hl.is_defined(ht.genomes_fafmax.faf95_max))
+                        & hl.is_defined(ht.exomes_fafmax.gnomad.faf95_max),
+                        "both",
+                    )
+                    .when(hl.is_defined(ht.genomes_fafmax.faf95_max), "genomes")
+                    .when(hl.is_defined(ht.exomes_fafmax.gnomad.faf95_max), "exomes")
+                    .default(hl.missing(hl.tstr)),
```

Comment:
also same question about validity checks

Reply:
even more for fafmax, do we really a validity check?

Reply:
the point of validity checks is to make sure nothing went awry, so we would want to know if this annotation is missing more often than expected given the missing rate of the faf annotations (and it sounds like this is taken care of based on Mike's comment in the other thread)